samples/OpenIdConnectSample/Readme.md

```diff
@@ -0,0 +1,44 @@
+# How to set up the sample locally
+
+The OpenIdConnect sample support multilpe authentication providers. In this instruction, we will explore how to set up this sample with both Azure Active Directory and Google Firebase Authentication
+
+## Determine your development environment and a few key variables
+
+If you are running the application with Visual Studio, launch settings are preconfigured in `launchSettings.json`. Your web application's URL is __https://localhost:44318__
+
+If you are running the application from command line or terminal, the launch settings are not binding to the `launch.json`. In this case the web application`s URL is __http://localhost:5000__. In addition to the different sample URL, environment variable ASPNETCORE_ENVIRONMENT should be set to DEVELOPMENT to enable user secret.
+
+## Configure the Authorization server
+
+### Configure with Azure Active Directory
+
+1. Set up a new Azure Active Directory (AAD) in your Azure Subscription.
+2. Open the newly created AAD in Azure web portal
+3. Navigate to the Applications tab
+4. Add a new Application to the AAD. Set the "Sign-on URL" to sample application's URL.
+5. Naigate to the Application, and click the Configure tab.
+6. Find and save the "Client Id".
+7. Add a new key in the "Keys" section. Save value of the key, which is the "Client Secret".
```

Comment:
AAD doesn't require you to set the full need to set the full path to it. And it fill in the reply URL automatically based on the sign on url.